juniper/src/integrations/chrono.rs

```diff
@@ -157,17 +151,14 @@ mod test {
 
     #[test]
     fn naivedate_from_input_value() {
-        naivedate_test("1996-12-19T16:39:57-08:00", 1996, 12, 19);
+        naivedate_test("1996-12-19", 1996, 12, 19);
     }
 
-    #[test]
-    fn naivedate_from_input_value_with_fractional_seconds() {
-        naivedate_test("1996-12-19T16:39:57.005-08:00", 1996, 12, 19);
-    }
-
-    #[test]
-    fn naivedate_from_input_value_with_z_timezone() {
-        naivedate_test("1996-12-19T16:39:57Z", 1996, 12, 19);
+    fn naivedate_serialization() {
+        let date = NaiveDate::from_ymd(2000, 1, 2);
+        let value = date.resolve();
+        let expected = Value::string("2000-01-02".to_string());
+        assert_eq!(value, expected);
```

Comment:
@LegNeato there is a test above that tests that given an input value it parses correctly and returns a NaiveDate. This https://github.com/graphql-rust/juniper/blob/master/juniper/src/integrations/url.rs#L23 looks similar.

I would like a test that given a NaiveDate it tests that it serialises correctly. Something that calls the `resolve` function to serialize. I haven't found an example test that do this.

I'm the app I'm building I'm getting a panic when trying to serialize NaiveDate, unsure what is wrong, but knowing how to test the serialization will help to find the issue.